src/modules/job-exec/Makefile.am

```diff
@@ -58,7 +58,7 @@ test_ldadd = \
 	$(top_builddir)/src/common/libtap/libtap.la \
 	$(top_builddir)/src/common/libflux-internal.la \
 	$(top_builddir)/src/common/libflux-core.la \
-	$(LIBPTHREAD) $(JANSSON_LIBS)
+	$(LIBPTHREAD)
```

Comment:
job-exec uses jansson directly, but is missing a `$(JANSSON_LIBS)` in its LIBADD.